en/file_formats/plan.md

```diff
@@ -98,40 +148,246 @@ A complex item is a higher level encapsulation of multiple MISSION_ITEMS treated
     ...
 },
 ```
-Complex items have two additional values associated with them:
+
+Complex items have these values associated with them:
 
 Key | Description
-----|------------
-complexItemType | Specifies the type of complex item. QGroundControl currently supports the following types: [survey](../file_formats/survey.md), fwLandingPattern
-version | Specifies the version for this complex item.
+--- | ---
+`TransectStyleComplexItem` | ?
+`type` | `ComplexItem` for a complex item
+`complexItemType` | The type of complex item. *QGroundControl* currently supports the following types: [survey](../file_formats/survey.md), fwLandingPattern
+`version` | The version of the complex item specification. Here we document version 4.
+`angle` | 
```

Comment:
Can you help explain what all the other fields are? Below this I included a cut down example (in HTML comments) that you might comment against.

Reply:
Yes.

Reply:
* Angle is the grid in degrees
* Entry location is the entry point into the survey (0, 1, 2, 3). Which are the corners of the rect with encompasses the survey. They don't really correspond to specific corners. You just pick one and see if it is what you want.
* Fly alternate transects allows you to skip every other transect when flying and then come back and fly the ones you missed. Use for fixed wing when the turnaround is too acute such that the vehicle can't make the turn. Hence you fly every other which makes the turaround larger